samples/HttpServer_Plugins/include/ContentDecoderPlugin.h

```diff
@@ -0,0 +1,82 @@
+/****
+ * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
+ * Created 2015 by Skurydin Alexey
+ * http://github.com/SmingHub/Sming
+ * All files of the Sming Core are provided under the LGPL v3 license.
+ *
+ *
+ * @author: 2021 - Slavey Karadzhov <slaff@attachix.com>
+ *
+ ****/
+
+#pragma once
+
+#include <Network/Http/Resource/HttpResourcePlugin.h>
+#include <Data/WebHelpers/base64.h>
+
+namespace
+{
+constexpr char ENCODING_NAME[] = "test";
+}
+
```

Comment:
Don't use anonymous namespaces in headers!

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf21-dont-use-an-unnamed-anonymous-namespace-in-a-header